### DISCUSSION TITLE
Edu Email is not provide by university

### DISCUSSION THREAD

**Discussion by sdanishka** [community/community#122316] [GitHub Education] [answered]
### Hi everyone,

My university does not provide an edu email how would I be still able to apply for a student pack

**Comment by Avotexs** [comment]
u can talk to the support and explain ue situation https://support.github.com/.

**Comment by Avotexs** [comment]
here exactly : https://support.github.com/request

**Comment by tstran155** [comment, marked as answer]
Hi sdanishka,

Here is another route you may consider.

- Create a GitHub account using whatever email address you have.
- Visit the GitHub student developer pack page (https://education.github.com/pack) and click on the “Get your pack” button.
- Fill out the application form. When it asks for proof of enrollment, upload the documentation you’ve prepared (student ID, official enrollment, registration letters or any document issued by your university that has your name). 

While an .edu email might speed up the application process due to easier verification, Github do accept other forms of proof.

Good luck!

**Reply by sdanishka** [reply]
Thank you very much brother mean a lot

**Reply by Sabiecodingnoob** [reply]
Hi it won't work on me, do you know something that I can do

**Reply by Munazakhursheed** [reply]
MINE NOT working too
<img width="1600" height="1200" alt="github" src="https://github.com/user-attachments/assets/367c7bb6-314a-4419-81ab-d80732c14b24" />

**Comment by iamdulanga** [comment]
@sdanishka 

can you please explain me how did you verified your account without a edu email.

**Reply by sdanishka** [reply]
The answer I selected (given by @tstran155) work for me

**Reply by iamdulanga** [reply]
ok.. i will try that.. thanks you..

**Comment by SumairAziz** [comment]
it doesn't work anymore, the submit option is greyed out, can't go any further it seems without providing a specific format email address.
